src/ExporterInterface.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sonata\Exporter;
+
+use Symfony\Component\HttpFoundation\StreamedResponse;
+
+interface ExporterInterface
+{
+    public function getAvailableFormats(): array;
```

Comment:
```suggestion
    /**
     * @return string[]
     */
    public function getAvailableFormats(): array;
```
to avoid the PHPStan error